libminifi/include/utils/ValueUtils.h

```diff
@@ -0,0 +1,193 @@
+/**
```

Comment:
I suggest renaming the file to `ValueParser.h` to reflect the class name. -Utils is usually used for potentially related, but largely independent set of utilities. This is a stateful parser with its own invariants

Reply:
done